tests/k8s/1-025_validate_resource_ignore_differences/01-argocd-with-resource-ignore-difference.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: argoproj.io/v1alpha1
```

Comment:
@reginapizza I think we should name this file `02-argocd-with-resource-ignore-difference` instead
kuttl uses the naming of the file as indication for the order in which the steps should be carried out 

01-whatever is followed by 01-assert, followed by 02-whatever followed by 02-assert for eg

right now it seems like there are multiple asserts for the same step